docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.9"
 services:
   anvil:
     image: ghcr.io/foundry-rs/foundry:latest
-    profiles: ["main", "setup"]
+    profiles: ["local"]
```

Comment:
When running with ENVIRONMENT=local `docker-compose up` will launch anvil locally. When running with ENVIRONMENT=development it will not launch the local anvil service